pkg/volume/rbd/rbd_util.go

```diff
@@ -55,38 +55,50 @@ const (
 	rbdImageWatcherSteps     = 10
 )
 
-// search /sys/bus for rbd device that matches given pool and image
 func getDevFromImageAndPool(pool, image string) (string, bool) {
```

Comment:
Would it not be more efficient to pass what to look for (rbd or nbd)? This way you always run getRBD even you really need only getNBD.

Reply:
This method is called only from other other location, and keeping this method helps hide some of the details from the caller.